src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -148,8 +159,8 @@ internal class RazorSemanticTokensLegend
             RazorDirectiveAttributeType,
             RazorDirectiveType,
             RazorCommentType,
-            RazorCommentTransitionType,
-            RazorCommentStarType, // 90
+            RazorCommentTransitionType, // 100
```

Comment:
These were mostly for helping me back when I was doing a lot more of this by hand back in the day, if the comment numbers are annoying to maintain feel free to destroy them.